code/modules/antagonists/wizard/equipment/spellbook.dm

```diff
@@ -139,10 +139,7 @@
 	spell_type = /obj/effect/proc_holder/spell/targeted/conjure_item/spellpacket
 	category = "Defensive"
 
-/datum/spellbook_entry/timestop
-	name = "Time Stop"
-	spell_type = /obj/effect/proc_holder/spell/aoe_turf/conjure/timestop
-	category = "Defensive"
+//timestop used to be here - defunct with new holopara system.

```

Comment:
you can't get timestop with `/obj/item/guardiancreator/wizard`

Reply:
true. Ill re-enable it if qwerty tells me to, but im a bit iffy enabling it for wizard considering how destructive the new stands are (frenzy + timestop = death)

Reply:
also, consider this. Timestop + the hand

literally able to teleport people inside the timestop field so you can abuse them infinitely because the hand

Reply:
on hippie I HAVE used timestop + sing hammer + the hand. it's fun lol.